packages/app/src/screens/AccountManageScreen.tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import { StyleSheet, View } from 'react-native';
+import { SvgXml } from 'react-native-svg';
+
+import { kakao_small_logo } from 'icons';
+import { Button, ListContainer, Stack, Text } from 'ui';
+import { colors } from 'design-token';
+
+const AccountManageScreen = () => {
+  return (
+    <View style={styles.wrapper}>
+      <Stack>
+        <ListContainer text='이메일 주소' subText='sadfsdfsd@naver.com' />
+        <ListContainer text='연결된 서비스' subText='카카오 계정 연결됨'>
+          <SvgXml xml={kakao_small_logo} />
+        </ListContainer>
+      </Stack>
+      <View style={styles.button_box}>
+        <View style={styles.button}>
+          <Button backgroundColor={colors.grayMedium} onPress={() => {}}>
+            <Text fontType='BOLD_16' color={colors.grayDark}>
+              회원 탈퇴
+            </Text>
+          </Button>
+        </View>
+        <View style={styles.gap} />
+        <View style={styles.button}>
+          <Button backgroundColor={colors.grayMedium} onPress={() => {}}>
+            <Text fontType='BOLD_16' color={colors.grayDark}>
+              로그아웃
+            </Text>
+          </Button>
+        </View>
+      </View>
+    </View>
+  );
+};
+
+const styles = StyleSheet.create({
+  wrapper: {
+    display: 'flex',
+    backgroundColor: colors.white,
+    height: '100%',
+    width: '100%',
+  },
+  button_box: {
+    position: 'absolute',
+    bottom: '5%',
+    width: '100%',
+    flexDirection: 'row',
+    alignItems: 'flex-end',
+    justifyContent: 'center',
+    paddingHorizontal: 20,
+  },
+  button: {
+    width: '50%',
+    justifyContent: 'center',
+    alignItems: 'center',
+  },
+  gap: {
+    width: 8,
+  },
+});
```

Comment:
```suggestion
const styles = StyleSheet.create({
  wrapper: {
    flex: 1,
    display: 'flex',
    backgroundColor: colors.white,
    justifyContent: 'space-between',
  },
  buttonBox: {
    width: '100%',
    flexDirection: 'row',
    justifyContent: 'center',
    paddingHorizontal: 20,
    paddingBottom: getBottomSpace(),
  },
  button: {
    flex: 1,
    justifyContent: 'center',
    alignItems: 'center',
  },
  gap: {
    width: 8,
  },
});
```

이렇게 변경하는 게 어떨까요?
`View`의 경우에는 width 디폴트가 100%라서 해줄 필요 없고, 이건 스크린컴포넌트니까 height를 100%하는 것보다 flex를 사용하는 게 더 좋아보입니다 ㅎㅎ
위에 `Stack`으로 감싸져 있으니 버튼을 내리려고 `space-between` 사용했구요~!
변수 이름은,, 카멜케이스로 통일부탁드려용

`getBottomSpace()`는 [이 패키지](https://github.com/ptelad/react-native-iphone-x-helper#readme)에 있는 함수인데 제가 이 후 PR에서 ui 모듈에 추가해놓은 상태입니다.
ui보다는 app에 있는 게 더 좋을 것 같아서, 경상님이 이거 추가하시고 제가 이후 PR에서 `peerDependencies`로 변경하겠습니다.

app에서 
```
yarn add react-native-iphone-x-helper
```
해주시면 될 것 같아요~


추가로, 아래 `PolicyScreen`에도 여기 코멘트에 적용되는 게 있는 것 같으니 확인부탁드립니다.

Reply:
자세한 리뷰 감사합니다! 많이배우네요! 

근데 혹시 `getBottomSpace()` 이거는 아이폰쪽 관련 패키지인것같은데, 혹시 해당패키지로 적용해도 안드로이드에 문제없는게 정상일까요?
제껄로 테스트했을때는 안드로이드폰에서는 버튼 아래쪽이 좀 잘리는 것 같더라구요.

Reply:
엇 공식 리드미에는 iphoneX 이상일 경우에만 적용이되고 나머지는 0으로 적용된다고 기재되어 있었는데, 아닌가요? 제가 이제까지 했을 때는 이상이 없어는데..! 다시 확인해보겠습니다 :)

https://github.com/ptelad/react-native-iphone-x-helper#getbottomspace

Reply:
앗 당연한거네요 ㅋㅋㅋ
`getBottomSpace()`는 아이폰 노치가 있는 경우에 하단 바가 있어서 그 공간 만큼만 띄워주는 거라
```ts
    paddingBottom: Platform.OS === 'android' ? 20 : getBottomSpace(),
```

이렇게 작성하면 될 것 같습니다!

<img src="https://user-images.githubusercontent.com/49540564/146625005-ec3953dd-e715-4706-9261-f9ccd4b5625b.png" width=400/>

Reply:
좋습니다~~